### ISSUE TITLE
[FEATURE] Support for Private Datasets when importing using the UI

### ISSUE THREAD

**Issue by alielfilali01** [recognai/rubrix#5671]
Request to add support for private datasets in the Hub for enhanced data privacy and control, as currently, only public datasets are supported for import from UI.

**Comment by AlbertoCastelo** [comment]
This would also be useful for me

**Comment by akimfromparis** [comment]
Agree, It would be great to better support private spaces and private datasets. Thank you! :bowing_man:
